src/System.Windows.Forms/src/System/Windows/Forms/ListView.cs

```diff
@@ -4822,6 +4856,12 @@ protected virtual void OnSearchForVirtualItem(SearchForVirtualItemEventArgs e)
         protected virtual void OnSelectedIndexChanged(EventArgs e)
         {
             ((EventHandler)Events[EVENT_SELECTEDINDEXCHANGED])?.Invoke(this, e);
+
+            if (this.SelectedItems.Count > 0 && this.SelectedItems[0].Focused)
+            {
+                var accessibilityObject = this.SelectedItems[0]?.AccessibilityObject;
```

Comment:
`SelectedItems[0]` can't be `null` here
```suggestion
                var accessibilityObject = SelectedItems[0].AccessibilityObject;
```

Don't use `var`